### DISCUSSION TITLE
Why you might not need a child theme

### DISCUSSION THREAD

**Discussion by karmatosed** [WordPress/developer-blog-content#195] [Topic Ideas]
Child themes were a practice often used previously and not that often needed with block-based themes. I want to create a post that outlines the things you can do within a block-based theme that previously you would have needed a child theme for.

This includes:
- Using patterns: or having more potential through them.
- Multiple styles through theme.json: there's a lot to include in this and the main section from fonts to how radically different you can create a theme just with changing the theme.json. Also how to use this feature.

There should be a section or note in this 'when to' for now which will include a note to not edit default themes and how to use Create Block Theme plugin.

**Comment by marybaum** [comment]
Love this!

**Comment by abhansnuk** [comment]
This will be a valuable article for readers at both beginners and intermediate level. Thanks Tammy for proposing it and willing to write it too.

**Comment by bph** [comment]
This discussion post is locked. Please continue the conversation moved to issue 
- #200
